.github/workflows/weekly_vuln_scan.yml

```diff
@@ -23,11 +22,37 @@ jobs:
   trivy:
     runs-on: ubuntu-latest
     steps:
-      - name: Run Trivy vulnerability scanner
-        uses: aquasecurity/trivy-action@master
-        with:
-          format: "table"
-          exit-code: "1"
-          ignore-unfixed: true
-          vuln-type: "os,library"
-          severity: "CRITICAL,HIGH"
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Check if Trivy is installed
```

Comment:
question: why are you installing it manually? They already provide images to run trivy: `aquasecurity/trivy-action@master`